assets/sass/legacy/_main.scss

```diff
@@ -323,11 +323,6 @@ collapsed due to narrow viewport) */
     display: inline-block;
 }
 
-.btn-group button:first-child {
-  border-top-right-radius: 0;
-  border-bottom-right-radius: 0;
-}
-
```

Comment:
This was removed because it caused the button at the right end of the widget to not have rounded corners. This change doesn't seem to have caused any effects in other places where `btn-group` was used.

Reply:
Indeed, removing it doesn't seem to affect anything. If `a-plus/exercise/templates/exercise/staff/analytics.html` is the only place that uses, there is an `<a>` element instead of `<button>` there.